app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -29,17 +29,16 @@ class CiYaml {
     }
     // Do not filter bringup targets. They are required for backward compatibility
     // with release candidate branches.
-    final Iterable<Target> totPresubmitTargets =
-        totConfig?._targets.where((Target target) => target.value.presubmit) ?? <Target>[];
-    final List<Target> totEnabledPresubmitTargets = _filterEnabledTargets(totPresubmitTargets);
-    totPresubmitTargetNames = totEnabledPresubmitTargets.map((Target target) => target.value.name).toList();
+    final Iterable<Target> totTargets = totConfig?._targets ?? <Target>[];
+    final List<Target> totEnabledTargets = _filterEnabledTargets(totTargets);
+    totTargetNames = totEnabledTargets.map((Target target) => target.value.name).toList();
     totPostsubmitTargetNames =
         totConfig?.postsubmitTargets.map((Target target) => target.value.name).toList() ?? <String>[];
   }
 
-  /// List of presubmit target names used to filter target from release candidate branches
+  /// List of  target names used to filter target from release candidate branches
```

Comment:
nit: two spaces between 'of' and 'target'.

Reply:
Removed.